specification/translation/data-plane/Azure.AI.DocumentTranslation/preview/2024-11-01-preview/openapi.json

```diff
@@ -0,0 +1,1537 @@
+            "$ref": "./examples/format.json"
+          }
+        }
+      }
+    },
+    "/document:translate": {
+      "post": {
+        "operationId": "DocumentTranslationOperations_DocumentTranslate",
+        "summary": "Submit a single document translation request to the Document Translation service",
+        "description": "Use this API to submit a single translation request to the Document Translation Service.",
+        "produces": [
+          "application/octet-stream",
+          "application/json"
+        ],
+        "consumes": [
+          "multipart/form-data"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/Azure.Core.Foundations.ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/DocumentTranslateBody.sourceLanguage"
+          },
+          {
+            "$ref": "#/parameters/DocumentTranslateBody.targetLanguage"
+          },
+          {
+            "$ref": "#/parameters/DocumentTranslateBody.category"
+          },
+          {
+            "$ref": "#/parameters/DocumentTranslateBody.allowFallback"
+          },
+          {
+            "$ref": "#/parameters/DocumentTranslateBody.translateTextWithinImage"
+          },
+          {
+            "$ref": "#/parameters/Azure.Core.ClientRequestIdHeader"
+          },
+          {
+            "name": "document",
+            "in": "formData",
+            "description": "Document to be translated in the form",
+            "required": true,
+            "type": "file"
+          },
+          {
+            "name": "glossary",
+            "in": "formData",
+            "description": "Glossary-translation memory will be used during translation in the form. ",
+            "required": false,
+            "type": "array",
+            "items": {
+              "type": "string",
+              "format": "binary"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "type": "file"
+            },
+            "headers": {
+              "x-metered-usage": {
+                "type": "integer",
+                "format": "int32",
+                "description": "Specifies consumption (the number of characters for which the user will be charged) for the translation job request"
+              },
+              "x-ms-client-request-id": {
+                "type": "string",
+                "format": "uuid",
+                "description": "An opaque, globally-unique, client-generated string identifier for the request."
+              },
+              "x-total-image-scans-failed": {
+                "type": "integer",
+                "format": "int32",
+                "description": "Specifies the number of failed image translations within a document translation job"
+              },
+              "x-total-image-scans-succeeded": {
```

Comment:
[⛔](https://aka.ms/azapi/guidelines#http-no-x-custom-headers) DO NOT use "x-" prefix for custom headers, unless the header already exists in production [[RFC 6648](https://datatracker.ietf.org/doc/html/rfc6648)].

Reply:
Updated to "total-image-scans-succeeded"